src/cpp/CMakeLists.txt

```diff
@@ -12,12 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
```

Comment:
I'm trusting the windows using developers on the correctness of the changes in this file. It all looks good to me, but I am not an expert on cmake + windows